system-tests/lib/cre/contracts/contracts.go

```diff
@@ -478,10 +480,22 @@ func ConfigureKeystone(input cre.ConfigureKeystoneInput) error {
 	}
 
 	evmChainsWithForwarders := make(map[uint64]struct{})
+	tronChainsWithForwarders := make(map[uint64]struct{})
 	for chainSelector, addresses := range allAddresses {
 		for _, typeAndVersion := range addresses {
 			if typeAndVersion.Type == keystone_changeset.KeystoneForwarder {
-				evmChainsWithForwarders[chainSelector] = struct{}{}
+				// Check if any of the blockchain outputs indicate this is a TRON chain
+				isTronChain := false
+				for _, bcOut := range input.BlockchainOutputs {
+					if bcOut.ChainSelector == chainSelector && bcOut.BlockchainOutput.Family == "tron" {
```

Comment:
nits: 
1. use `strings.EqualFold` instead of `==`? 
2. maybe there is a constant in the CTF or `cldf_tron` for "tron" string literal that we could use?